docs/design.md

```diff
@@ -873,23 +873,31 @@ replicated. In the absence of heartbeats, followers become candidates
 after randomized election timeouts and proceed to hold new leader
 elections. Cockroach weights random timeouts such that the replicas with
 shorter round trip times to peers are more likely to hold elections
-first. Although only the leader can propose a new write, and as such
-must be involved in any write to the consensus log, any replica can
-service reads if the read is for a timestamp which the replica knows is
-safe based on the last committed consensus write and the state of any
-pending transactions.
-
-Only the leader can propose a new write, but Cockroach accepts writes at
-any replica. The replica merely forwards the write to the leader.
-Instead of resending the write, the leader has only to acknowledge the
-write to the forwarding replica using a log sequence number, as though
-it were proposing it in the first place. The other replicas receive the
-full write as though the leader were the originator.
+first. 
+
+Only the leader can propose a consistent read or a new write, but
+Cockroach accepts read/write requests at any replica. The replica
```

Comment:
All of this forwarding isn't really there as far as I can tell, see the above comment.